backend/accessibility_templates/aa-007.xml

```diff
@@ -0,0 +1,13 @@
+<AccessibilityAttributes>
+ <DialogueEnhancementAttributes>
+  <AudioAttributes>
+   <Coding href="urn:mpeg:mpeg7:cs:AudioCodingFormatCS:2001:1"/>
```

Comment:
Although also MPEG schema is allowed...wouldn't it be better to use the DVB schema?

Reply:
The MPEG-7 CS term is 
````
<ClassificationScheme uri="urn:mpeg:mpeg7:cs:AudioCodingFormatCS:2001" domain="//MediaInformation/MediaProfile/MediaFormat/AudioCoding/Format">
    <Term termID="1">
        <Name xml:lang="en">AC3</Name>
        <Definition xml:lang="en">Dolby AC3</Definition>
    </Term>
````
and the definition of the `<AudioAttributes><Coding>` in ETS TS 102 822-3-1 v1.13.1 says
> This term should be taken from the MPEG-7 "AudioCodingFormatCS" classification scheme listed in clause B.2.3 of ISO/IEC 15938-5 [2], i.e. ...

Indeed, as this is a **_should_** we can use other terms. Perhaps from the DVB AudioCodecCS we can use `urn:dvb:metadata:cs:AudioCodecCS:2020:3.1`...
````
	<Term termID="3">
		<Name>AC3</Name>
		<Term termID="3.1">
			<Name>E-AC3</Name>
			<Definition>Dolby Enhanced AC3</Definition>
		</Term>
	</Term>
````

Reply:
A177r6 says in Table 18:

> The attributes of any audio component in the service. Only leaf term values from the following classification schemes are permitted values for Coding@href (see also note 1)
> •	urn:dvb:metadata:cs:AudioCodecCS:2007
> •	urn:dvb:metadata:cs:AudioCodecCS:2020
> •	urn:mpeg:mpeg7:cs:AudioCodingFormatCS:2001

The examples in A177r6 only use the DVB CS.